src/main/java/com/programmers/VoucherManagementApplication/voucher/Voucher.java

```diff
@@ -0,0 +1,59 @@
+package com.programmers.VoucherManagementApplication.voucher;
+
+import com.programmers.VoucherManagementApplication.vo.Amount;
+import com.programmers.VoucherManagementApplication.vo.VoucherType;
+
+import java.util.UUID;
+
+public abstract class Voucher {
+
+    private final UUID voucherId;
+    private final VoucherType voucherType;
+    private final Amount amount;
+    private String usageStatus = "N";
+
+    public Voucher(UUID voucherId, VoucherType voucherType, Amount amount) {
```

Comment:
```suggestion
    protected Voucher(UUID voucherId, VoucherType voucherType, Amount amount) {
```
추상 클래스의 생성자는 protected를 기본으로 사용합니다.